capture/engine_scripts/puppet/clickAndHoverHelper.js

```diff
@@ -16,6 +17,15 @@ module.exports = async (page, scenario) => {
       await page.click(clickSelectorIndex);
     }
   }
+  
+  if (scrollToSelector) {
+    for (const scrollToSelectorIndex of [].concat(scrollToSelector)) {
```

Comment:
Do you intend `scrollToSelector` to take an array value?  or just a string? Array feels like it would produce some noisy results.